src/core/protocol-authorization.ts

```diff
@@ -254,26 +253,26 @@ export class ProtocolAuthorization {
     const recordsWriteMessage = inboundMessage as RecordsWriteMessage;
 
     const protocolPath = recordsWriteMessage.descriptor.protocolPath!;
-    const recordDefinitionId = ProtocolAuthorization.getRecordDefinitionId(protocolPath);
-    // existence of recordDefinition has already been verified
-    const recordDefinition: ProtocolRecordDefinition = Protocols.getRecordDefinition(protocolDefinition, recordDefinitionId)!;
+    const typeName = ProtocolAuthorization.getTypeId(protocolPath);
```

Comment:
I guess concept of ID no longer applies?
Consider:

```suggestion
    const typeName = ProtocolAuthorization.getTypeName(protocolPath);
```
or
```suggestion
    const typeName = ProtocolAuthorization.getType(protocolPath);
```

Reply:
update to `getTypeName`